prow/build_test_update.md

```diff
@@ -4,61 +4,56 @@ This guide is directed at Prow developers and maintainers who want to build/test
 
 ## How to build and test Prow
 
-You can build, test, and deploy Prow’s binaries, container images, and cluster resources using [`bazel`](https://bazel.build).
+You can build, test, and deploy Prow’s binaries, container images, and cluster resources.
 
-Build with:
+Build locally with:
 ```shell
-bazel build //prow/...
+make -C prow build-images
+```
+Push to remote with
+```shell
+make -C prow push-images REGISTRY=<YOUR_REGISTRY>
 ```
 Unit test with:
 ```shell
-bazel test --@io_bazel_rules_go//go/config:race //prow/...
+make tes
 ```
 Integration test with([more details](./test/integration)):
 ```shell
 ./prow/test/integration/integration-test.sh
 ```
 Individual packages and components can be built and tested like:
 ```shell
-bazel build //prow/cmd/hook
-bazel test //prow/plugins/lgtm:go_default_test
+go build ./prow/cmd/hook
+go test ./prow/plugins/lgtm
 ```
+(Note: this doesn't work with deck as it requires some static files, use
+[`runlocal`](/prow/cmd/deck/runlocal) if desired.)
```

Comment:
This sounds like it is suggesting that it is no longer possible to run unit tests on deck, but hopefully that is not the case. If `go build` and `go test` are not sufficient for running deck's unit tests, please include the necessary commands. (runlocal  is not equivalent)

Reply:
unit tests of deck can run with `go test`. The test of static files is done by integration test which includes compiling typescripts. I have updated this part